### PR TITLE
COL-1461 Schema SQL for remaining SuiteC tables

### PR DESCRIPTION
**Pull request by pauline2k**
https://jira.ets.berkeley.edu/jira/browse/COL-1461

### CODE REVIEW

File: scripts/db/schema.sql [hunk 1]
```diff
@@ -66,25 +190,23 @@ CREATE TABLE assets (
     source character varying(255),
     thumbnail_url character varying(255),
     title character varying(255),
-    trending_percentile integer DEFAULT 0 NOT NULL,
-    trending_score integer DEFAULT 0 NOT NULL,
```
Comment (pauline2k) [inline scripts/db/schema.sql:70]: Squiggy, sadly, will not be measuring Impact.

File: scripts/db/schema.sql [hunk 2]
```diff
@@ -66,25 +190,23 @@ CREATE TABLE assets (
...
     type enum_assets_type NOT NULL,
     url character varying(255),
     views integer DEFAULT 0,
     visible boolean DEFAULT true NOT NULL,
     created_at TIMESTAMP WITH TIME ZONE NOT NULL,
     updated_at TIMESTAMP WITH TIME ZONE NOT NULL
 );
-ALTER TABLE assets OWNER TO squiggy;
```
Comment (pauline2k) [inline scripts/db/schema.sql:78]: I'm experimenting to see if we can get away without using these hardcoded ownership statements in the schema. The actual app user might be called "squiggy" or "app_squiggy" depending on environment.

File: scripts/db/schema.sql [hunk 3]
```diff
@@ -45,6 +72,105 @@ CREATE TYPE enum_assets_type AS ENUM (
...
+    object_type enum_activities_object_type NOT NULL,
+    metadata json,
+    asset_id integer,
+    course_id integer NOT NULL,
+    user_id integer NOT NULL,
+    actor_id integer,
+    reciprocal_id integer,
+    created_at timestamp with time zone NOT NULL,
+    updated_at timestamp with time zone NOT NULL
+);
```
Comment (pauline2k) [inline scripts/db/schema.sql:98]: The "activities" table hangs around for engagement index purposes. The "events" table will not be implemented.

**Comment by johncrossman** [comment]
Looks right, but it ain't Impactful. Thanks @pkerschen

**Pull request merged**